polling_stations/apps/addressbase/models.py

```diff
@@ -18,6 +27,59 @@ def points_for_postcode(self, postcode):
         qs = qs.values_list("location", flat=True)
         return list(qs)
 
+    def get_uprn(self, uprn):
+        cursor = connection.cursor()
+        cursor.execute(
+            """
+                SELECT a.address, a.postcode, u.lad, u.uprn, u.polling_station_id, a.location
+                FROM addressbase_address a
+                    JOIN addressbase_uprntocouncil u
+                    ON a.uprn = u.uprn
+                WHERE a.uprn=%s
+            """,
+            [uprn],
+        )
+        row = cursor.fetchone()
+        if row:
+            return UprnInfo(
+                address=row[0],
+                postcode=row[1],
+                council=row[2],
+                uprn=row[3],
+                polling_station_id=row[4].replace(" ", ""),
```

Comment:
Is this actually a good idea? What's it trying to guard against?
If a station ID has a space in it by this point then presumably it does in the stations table too so this might not really be a good idea. I think if we want to ensure IDs don't have a space in them (for whatever reason) we should be doing that in the import scripts/base importers and then at the point the IDs are in the DB we assume they should already be clean.

Reply:
I'm now not sure what this was doing, but taking it out doesn't break anything I can see. So probably shouldn't be there.